cmd/neutrinod/main.go

```diff
@@ -0,0 +1,295 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/gorilla/websocket"
+	log "github.com/sirupsen/logrus"
+	"github.com/vulpemventures/neutrino-elements/pkg/blockservice"
+	"github.com/vulpemventures/neutrino-elements/pkg/node"
+	"github.com/vulpemventures/neutrino-elements/pkg/protocol"
+	"github.com/vulpemventures/neutrino-elements/pkg/repository/inmemory"
+	"github.com/vulpemventures/neutrino-elements/pkg/scanner"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+const (
+	writeWait       = 10 * time.Second
+	pongWait        = 60 * time.Second
+	pingPeriod      = (pongWait * 9) / 10
+	maxMessageSize  = 512
+	shutdownTimeout = 2 * time.Second
+
+	unspents EventType = "UNSPENT"
+)
+
+var (
+	newline = []byte{'\n'}
+	space   = []byte{' '}
+)
+
+func main() {
+	repoFilter := inmemory.NewFilterInmemory()
+	repoHeader := inmemory.NewHeaderInmemory()
+
+	nodeCfg := node.NodeConfig{
+		Network:        "nigiri",
+		UserAgent:      "neutrino-elements:test",
+		FiltersDB:      repoFilter,
+		BlockHeadersDB: repoHeader,
+	}
+
+	blockSvc := blockservice.NewEsploraBlockService("http://localhost:3001")
+
+	elementsNeutrinoDaemon, err := NewElementsNeutrinoServer(
+		nodeCfg,
+		blockSvc,
+		"localhost:18886",
+		"localhost:8080",
+	)
```

Comment:
shall we parametrize this with config through flags/envs

Reply:
Done